modules/simple_forms_api/app/services/simple_forms_api/s3_service/archive_submission_to_pdf.rb

```diff
@@ -0,0 +1,184 @@
+# frozen_string_literal: true
+
+# To use
+# ids = <array of submission ids to archive>
+# parent_dir = <the name of the s3 'folder' where these archives will be put>
+#
+# to see your archive in s3
+# 1. go here https://console.amazonaws-us-gov.com/s3/home?region=us-gov-west-1#
+# 2. login with 2fa
+# 3. search for dsva-vetsgov-prod-reports
```

Comment:
I know we've been using this, but if we are standardizing and trying to implement something to be widely used, is it worth trying to evaluate if this is actually where we are supposed to be putting these? Or if we wanted a new bucket specifically blessed for this?